tests/test_view_count.py

```
@@ -0,0 +1,185 @@
+import pytest
+import concurrent.futures
+import time
+from app.services.post import PostService
+from app.schemas.post import PostCreate
+from app.db.database import SessionLocal
```

Ruff (0.11.9)

`app.db.database.SessionLocal` imported but unused

Remove unused import: `app.db.database.SessionLocal`

(F401)